amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -132,7 +132,12 @@ export class DashboardPage extends React.Component<
     }
 
     tabInfo.push({
-      content: <QueryList queries={this.props.dashboard.queries} />,
+      content: (
+        <QueryList
+          product={this.props.dashboard.product}
```

Comment:
i c, so backend returns product info actually? I recalled we don't store product node in the graph which is a good extension.

Reply:
It's stored on the `dashboard` node. And so because we only have this component in the context of `DashboardPage` which has the information for the dashboard resource, we can get away with passing `product` down to all of the children components like the `QueryList` and `QueryListItem`. 

If we ever wanted to show a `QueryListItem` outside of the `DashboardPage` context, then that is when we absolutely need the backend to return the `product` with the query. However, if we wanted to make that extension, it wouldn't hurt at all and would make this code slightly cleaner by being able to delete most of these changes.